logforwarder/src/apps/vespa-logforwarder-start/child-handler.cpp

```diff
@@ -76,14 +76,15 @@ void
 ChildHandler::startChild(const vespalib::string &prefix)
 {
     if (! _childRunning) {
+        // it is possible that splunk was already running anyway, so
+        // make sure we restart it to get new config activated:
+        runSplunk(prefix, {"stop"});
+        sleep(1);
```

Comment:
It looked from the logs like stopping `splunkd` would take roughly 15s. I don't know what happens if we don't wait for the previous instance to shut down, but it's probably OK, since we don't know otherwise by now.